.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,78 @@
+name: Build and Test PR
+
+on:
+  pull_request: # Trigger for pull requests.
+    types: [opened, synchronize, reopened]
+  workflow_dispatch: # Allows for manual triggering.
+    inputs:
+      ref:
+        description: "The ref to build and test."
+        required: False
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: src
```

Comment:
This is an artifact of the Packager workflow, where source needs to be in a specific subfolder with this name.  Here, you should omit it and use the default (repo name).